python/ctsm/modify_fsurdat/modify_fsurdat.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, my_data, lon_1, lon_2, lat_1, lat_2, landmask_file):
 
         self.file = my_data
 
-        self.not_rectangle = self._get_not_rectangle(
+        self.rectangle = self._get_rectangle(
```

Comment:
New function `get_dom_cft` needs `rectangle` rather than `not_rectangle` so I did a minor rewrite here and a few lines down.